docs/pages/base/api/table-pagination-unstyled.json

```diff
@@ -43,6 +43,6 @@
   "forwardsRefTo": "HTMLTableCellElement",
   "filename": "/packages/mui-base/src/TablePaginationUnstyled/TablePaginationUnstyled.tsx",
   "inheritance": null,
-  "demos": "<ul><li><a href=\"/material-ui/react-table/\">Tables</a></li></ul>",
+  "demos": "<ul><li><a href=\"/base/components/table-pagination/\">Table Pagination</a></li>\n<li><a href=\"/material-ui/react-table/\">Tables</a></li></ul>",
```

Comment:
This seems wrong. 🤔

Reply:
@michaldudak I think you have to remove `TablePaginationUnstyled` in the markdown header from `docs/data/material/components/tables/tables.md`

Reply:
I must have forgotten about this. Thanks!